Makefile

```diff
@@ -50,6 +52,13 @@ woke-install:
 	@type woke >/dev/null 2>&1 || \
             { echo "Installing \"woke\" snap... \n"; sudo snap install woke; }
 
+pa11y-install:
+	@type $(PA11Y) >/dev/null 2>&1 || { \
+			echo "Installing \"pa11y\" from npm... \n"; \
+			mkdir -p $(SPHINXDIR)/node_modules/ ; \
+			npm install --prefix $(SPHINXDIR) pa11y; \
+		}
```

Comment:
```
Installing "pa11y" from npm... 

/bin/sh: 4: npm: not found
make: *** [Makefile:56: pa11y-install] Error 127
```

We could add some checks for that and give a different error message, but I don't think it's required.
But can we assume that npm is installed on GitHub servers?